src/EditorFeatures/Core/Commands/GoToPreviousMethodCommandArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Commands
+{
+    [ExcludeFromCodeCoverage]
+    internal class GoToPreviousMethodCommandArgs : CommandArgs
```

Comment:
Okay, renamed everything to GoToAdjacentMember except the `IOleCommandTarget` stuff.